org.opent2t.onboarding.insteon/js/common.js

```diff
@@ -0,0 +1,23 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
```

Comment:
Removed.

---
In reply to: [82906938](https://github.com/openT2T/onboarding/pull/21#discussion_r82906938) [](ancestors = 82906938)